x-pack/plugins/osquery/common/config.ts

```diff
@@ -10,6 +10,7 @@ import { schema } from '@kbn/config-schema';
 
 export const ConfigSchema = schema.object({
   actionEnabled: schema.boolean({ defaultValue: false }),
+  enabled: schema.boolean({ defaultValue: true }),
```

Comment:
I've a few questions. 

From the linked PR it looks like this sort of flag is nice to have if you want to disable this plugin on serverless? Is that why this is needed now? The PR description doesn't say much 😅 

Also, I would generally prefer not having generic names, a better name would be `isPluginEnabled`. Perhaps also a good idea to add a comment as to what this flag does if it is named a generic `enabled`.

Reply:
Hey Ash! You're right about the serverless part, this is the aim of this PR :) 
Re the naming: 'enabled' is used across the projects, so I am going to use the same, hope you're ok with it 👍 
Thanks for the approval!